lib/ansible/module_utils/network/dcnm/dcnm.py

```diff
@@ -45,8 +45,22 @@ def validate_list_of_dicts(param_list, spec):
                 type = spec[param].get('type')
                 if type == 'str':
                     item = v.check_type_str(item)
+                    if spec[param].get('length_max'):
+                        if 1 <= len(item) <= spec[param].get('length_max'):
```

Comment:
We should chat about how we want to handle custom checks like this.  My only concern is that this list could grow over time and we should come up with a way to list all of the possible custom parameters in one place along with any user validate error messages we want to display.

Reply:
Yeah, understand, will leave it open for now.